src/app/personside/infotabs/varsel/Varsel.tsx

```diff
@@ -97,99 +102,91 @@ function getVarselTekst(varsel: VarselModell) {
+
     return (
         <li>
             <StyledPanel>
                 <article aria-labelledby={tittelId.current}>
                     <HeaderStyle onClick={toggleOpen}>
-                        <Normaltekst>{formaterDato(varsel.forstBehandlet)}</Normaltekst>
-                        <Element id={tittelId.current} tag="h4">
-                            {varsel.tekst}
-                        </Element>
+                        <Normaltekst>{datoer}</Normaltekst>
+                        <EllipsisElement id={tittelId.current} tag="h4">
+                            {props.tittel}
+                        </EllipsisElement>
                         <Kommaliste aria-label="Kommunikasjonskanaler">
-                            <Normaltekst tag="li">NOTIFIKASJON {aktiv}</Normaltekst>
+                            {props.kanaler.map((kanal) => (
+                                <Normaltekst tag="li" key={kanal}>
+                                    {kanal}
+                                </Normaltekst>
+                            ))}
                         </Kommaliste>
                         <VisMerChevron
                             focusOnRelativeParent={true}
                             onClick={toggleOpen}
                             open={open}
-                            title={(open ? 'Skjul' : 'Vis') + ' mer informasjon om notifikasjon'}
+                            title={(open ? 'Skjul' : 'Vis') + ' mer informasjon om varsel/notifikasjon'}
                         />
                     </HeaderStyle>
-                    <UnmountClosed isOpened={open}>
-                        <GraattDefinisjonsListe>
-                            <dt>Produsert av:</dt>
-                            <dd>{varsel.produsent}</dd>
-
-                            <dt>Tekst:</dt>
-                            <dd>{varsel.tekst}</dd>
-
-                            <dt>Link:</dt>
-                            <dd>{varsel.link}</dd>
-                        </GraattDefinisjonsListe>
-                    </UnmountClosed>
+                    <UnmountClosed isOpened={open}>{props.children}</UnmountClosed>
                 </article>
             </StyledPanel>
         </li>
     );
 }
 
-function Varsel({ varsel }: { varsel: VarselModell }) {
-    const open = useAppState((state) => state.varsler.aapneVarsler).includes(varsel);
-    const dispatch = useDispatch();
-    const setOpen = (open: boolean) => dispatch(toggleVisVarsel(varsel, open));
-    const sortertMeldingsliste = varsel.meldingListe.sort(datoSynkende((melding) => melding.utsendingsTidspunkt));
-    const tittelId = useRef(guid());
-
-    const toggleOpen = () => setOpen(!open);
+function DittNavEventVarsel({ varsel }: { varsel: DittNavEvent }) {
+    const aktiv = varsel.aktiv ? '' : ' (Ferdigstilt)';
+    const datoer = [formaterDato(varsel.forstBehandlet)];
+    const tittel = `Notifikasjon${aktiv}: ${varsel.tekst}`;
+    const kanaler = ['DITT_NAV', ...varsel.eksternVarslingKanaler];
```

Comment:
Burde kanskje skrive det uten understrek? `DITT NAV`

Reply:
`DITT_NAV` er koden som vises for de gamle varslene, så tenker det er greit å holde det så likt som mulig.
Så kan vi evt lage en mapping over `kanaler` som er generisk for alle varsler om det blir noen utfordringer med det.